### ISSUE TITLE
Solltemperatur am HmIP-WTH-2 stellt sich in Home App zurück

### ISSUE THREAD

**Issue by rost2802** [thkl/hap-homematic#411]
Hallo,

ich hab da eine Problem mit dem Wandthermostat HmIP-WTH-2. Nach Änderung der Solltemperatur in der Home App stellt sich die Temperatur nach erneutem öffnen des Gerätes wieder zurück.

![71772724-47feee00-2f50-11ea-9336-81707e8fcf6e](https://user-images.githubusercontent.com/59236127/114286467-467b7a00-9a5f-11eb-87f0-32acfd6890c7.gif)

**Comment by rost2802** [comment]
Hier wurde das Problem schonmal gelöst, evtl. ist das eine Hilfe:

[https://github.com/rdmtc/RedMatic-HomeKit/issues/245(url)

**Comment by rost2802** [comment]
Hat das Problem sonst keiner? Oder hab ich was übersehen?

**Comment by rost2802** [comment]
Vielen Dank für deine Antwort, gesteuert wurde bereits über die Heizgruppe. Eben mal ein Gerät was direkt das WTH ansteuert getestet mit selbem fehlerhaftem Ergebnis. Gruß Ronny

**Comment by blackbasket** [comment]
Also bei mir reagiert (leider schon immer) die Home App nicht so schnell und ich habe keinen WTH zum Testen hier...

aber in einer Gruppe mit eTRVs kann ich es - direkt am Radiator - nachvollziehen... ebenso an einem einzelnen Radiator ohne Gruppe...

Das Verhalten tritt bei mir aber nur im Auto-Mode auf... im Manu-Mode nicht...

**Comment by rost2802** [comment]
Genau... auch bei mir nur im Auto-Mode das Problem. Wenn ich die Temperatur direkt am WTH verstelle und dann die HomeApp öffnen stellt sie sich auch wieder zurück sobald ich das Thermostat öffne.

**Comment by blackbasket** [comment]
Ja, das ist sehr ähnlich dem Verhalten in #378

**Comment by tornenen** [comment]
Ich habe das Problem auch. 
Gibt es schon eine Lösung?

**Comment by timwegenerDE** [comment]
Auch bei mir springt die Temperatur zurück. Wenn ich die Temperatur in einer Automation setze, wird diese ebenfalls nicht gesetzt. Ist das bei den anderen hier auch so?

**Comment by tornenen** [comment]
Ja war bei mir auch so.

Bin mittlerweile auf ioBroker gewechselt und bringe meine Geräte darüber ins HomeKit. Damit läuft es auch über Automationen etc.

**Comment by timwegenerDE** [comment]
Bei mir tritt das Problem tatsächlich nicht mehr auf. Bewusst etwas an HAP oder der CCU geändert habe ich nicht.

**Comment by blackbasket** [comment]
Ne, sieht gut aus @ChrWi ... Gerade getestet...
